src/components/Signup/Signup.js

```diff
@@ -127,6 +127,7 @@ export default injectIntl(function({ intl }) {
                 <PasswordFieldItem
                   fieldName={fieldNames.password}
                   label={_('signup.label_password')}
+                  showHideButton="true"
```

Comment:
What is this?

Reply:
Ahh, entiendo.

Pero creo que acá hay una incosistencia que deberíamos resolver.

Según entiendo, siempre tendríamos que permitir mostrar u ocultar el password.

@IgnacioRodrigues @GusBaamonde podrás revisar eso?

![image](https://user-images.githubusercontent.com/1157864/55483575-3b361e00-55fd-11e9-8829-f1b426b5bda5.png)

![image](https://user-images.githubusercontent.com/1157864/55483635-5d2fa080-55fd-11e9-99dd-0fdba5316dab.png)

BTW, por que le debemos recordar los requisitos de validación del password al usuario al momento de cargarla?

Esos requisitos son para definir el password, no para introducirlo.

Reply:
Si, deberia que poderse ver u ocultar la contraseñ, lo cierto es que en el mockup no esta de la misma forma. Lo reviso con Santiago, pero coincido con @andresmoschini que deberia haber consistencia.

Reply:
in login UI doesn't has the eye icon to show password, i add a custom parameter to hide or show the eye icon

Reply:
Ah me llego tarde los mensajes recien veo la actualizacion. Avisen que deciden hacer. yo lo oculte porque en el login no estaba. si hay que volverlo a poner lo agrego @IgnacioRodrigues @andresmoschini

Reply:
Señore @andresmoschini @IgnacioRodrigues @fcoronelmakingsense ! Recién hable con Santi por varios motivos. Incluí en la charla este tema. Esta de acuerdo que hay inconsistencia en este punto. A la tarde habla con Caro(creo que me dijo, es la chica que arma los contenidos) para ver si podemos acortar la frase, eso nos daría margen para colocar el ojito y mostrar la contraseña.

Reply:
Muchas gracias! avisen

Reply:
Ya hablé con Santi, dejemoslo como en Signup, ese texto no es necesario e incluso podría ser confuso si llegaran a cambiar los requisitos de los password.

@fcoronelmakingsense avisame si queda alguna duda.

Reply:
Listo, actualizado!